activestorage/lib/active_storage/engine.rb

```diff
@@ -109,6 +110,11 @@ class Engine < Rails::Engine # :nodoc:
     initializer "active_storage.verifier" do
       config.after_initialize do |app|
         ActiveStorage.verifier = app.message_verifier("ActiveStorage")
```

Comment:
Could we expose the `verifier` earlier and let users call `rotate` on it directly? Potentially `config.active_storage.verifier.rotate`? We'd need to see if there's other ways to set the verifier that we'd need to account for too.